ingests_api/src/test/scala/uk/ac/wellcome/platform/storage/ingests/api/IngestsApiFeatureTest.scala

```diff
@@ -288,17 +288,30 @@ class IngestsApiFeatureTest
                     )
                 }
 
-                val requests =
-                  listMessagesReceivedFromSNS(topic).map(messageInfo =>
+                // Archivist
+                val archivistRequests =
+                  listMessagesReceivedFromSNS(archivistTopic).map(messageInfo =>
                     fromJson[IngestBagRequest](messageInfo.message).get)
 
-                requests shouldBe List(
+                archivistRequests shouldBe List(
                   IngestBagRequest(
                     id,
                     storageSpace = StorageSpace(spaceName),
                     archiveCompleteCallbackUrl = Some(testCallbackUri),
                     zippedBagLocation = ObjectLocation("bucket", "key.txt")
                   ))
+
+                // Unpacker
+                val unpackerRequests =
+                  listMessagesReceivedFromSNS(unpackerTopic).map(messageInfo =>
+                    fromJson[UnpackRequest](messageInfo.message).get)
```

Comment:
minor: there's a neater way to do this. I forget the exact helper, but something like

```scala
assertSnsReceivesOnly(unpackerTopic, UnpackRequest(…))
```

Reply:
I think I'd prefer to keep this as is in this PR (because it's easier and because that changes the pattern used elsewhere in this file).